main.pm

```diff
@@ -340,7 +340,7 @@ sub is_reboot_after_installation_necessary() {
 sub load_inst_tests() {
     loadtest "installation/welcome.pm";
     if (!check_var('BACKEND', 'ipmi') && !check_var('BACKEND', 's390x') && !get_var('USBBOOT')) { # network installs in general, but we have no setting for it yet
-        loadtest "installation/check_medium.pm";
+        loadtest "installation/check_medium.pm" unless (get_var('NETBOOT'));
```

Comment:
For consistencies sake, please change this so it's implimented as "&& !get_var('NETBOOT')" as part of the if statement on line 342 instead of as an 'unless' here